presto-docs/src/main/sphinx/release/release-0.189.rst

```diff
@@ -0,0 +1,81 @@
+=============
+Release 0.189
+=============
+
+General Changes
+---------------
+
+* Fix query failure while logging query plan.
+* Fix client hang with ``LIMIT`` queries when ``optimizer.force-single-node-output``
+  is disabled.
+* Fix a bug in the :func:`bing_tile_at` and :func:`bing_tile_polygon` functions
+  that may produce incorrect results for points that are close to tile edges.
+* Fix scope semantics when lambda argument shadows relation column.
```

Comment:
maybe `Fix variable resolutions when lambda argument has the same name as a table column.`?  I'm not sure all users will understand the technical jargon.